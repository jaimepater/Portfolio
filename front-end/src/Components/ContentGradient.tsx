import styled from 'styled-components';

const ContentGradient = styled.div`
 background: linear-gradient(
  180deg, rgba(22,3,100,1) 0%, 
  rgba(45,93,154,1) 48.11%,
  rgba(48,103,160,1) 53.55%,
  rgba(53,121,171,1) 63.18%,
  rgba(59,146,186,1) 76.56%, 
  rgba(63,162,196,1) 85.35%,
  rgba(65,170,201,1) 89.54%, 
  rgba(70,189,213,1) 100%);
  height: 300%;
`;

export default ContentGradient;
