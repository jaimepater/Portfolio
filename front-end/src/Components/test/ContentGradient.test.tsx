import React from "react";
import { shallow } from 'enzyme';
import ContentGradient from "../ContentGradient";

describe('ContentGradient', () => {
  it('Renders ContentGradient', () => {
    const shallowComponent = shallow(<ContentGradient />);
    expect(shallowComponent).toMatchSnapshot();
  });
});