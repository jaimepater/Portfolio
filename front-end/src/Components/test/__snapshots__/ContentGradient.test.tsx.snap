// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentGradient Renders ContentGradient 1`] = `
<div
  className="sc-AykKC cKmjPM"
/>
`;
